src/styles/standardtable.scss

```diff
@@ -0,0 +1,8 @@
+table.os-raise-standardtable {
+  table,
```

Comment:
Is this actually necessary? (I think it would apply to a table within a table :man_shrugging: )